packages/dashboard-frontend/src/pages/Loader/Factory/__tests__/__snapshots__/applyingDevfileErrorItems.spec.tsx.snap

```diff
@@ -0,0 +1,98 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ApplyingDevfile error items should correctly render the component 1`] = `
+<div
+  className="pf-c-content"
+>
+  <small
+    className=""
+    data-pf-content={true}
+  >
+    The Devfile in the git repository is invalid:
+  </small>
+  <small
+    className=""
+    data-pf-content={true}
+  >
+    <div
+      className="pf-c-code-block error"
+    >
+      <div
+        className="pf-c-code-block__header"
+      >
+        <div
+          className="pf-c-code-block__actions"
+        >
+          <div
+            className="pf-c-code-block__actions-item"
+          >
+            <button
+              aria-disabled={false}
+              aria-label={null}
+              className="pf-c-button pf-m-link"
+              data-ouia-component-id="OUIA-Generated-Button-link-1"
+              data-ouia-component-type="PF4/Button"
+              data-ouia-safe={true}
+              disabled={false}
+              onClick={[Function]}
+              role={null}
+              type="button"
+            >
+              <svg
+                aria-hidden={true}
+                aria-labelledby={null}
+                fill="currentColor"
+                height="1em"
+                role="img"
+                style={
+                  Object {
+                    "verticalAlign": "-0.125em",
+                  }
+                }
+                viewBox="0 0 448 512"
+                width="1em"
+              >
+                <path
+                  d="M320 448v40c0 13.255-10.745 24-24 24H24c-13.255 0-24-10.745-24-24V120c0-13.255 10.745-24 24-24h72v296c0 30.879 25.121 56 56 56h168zm0-344V0H152c-13.255 0-24 10.745-24 24v368c0 13.255 10.745 24 24 24h272c13.255 0 24-10.745 24-24V128H344c-13.2 0-24-10.8-24-24zm120.971-31.029L375.029 7.029A24 24 0 0 0 358.059 0H352v96h96v-6.059a24 24 0 0 0-7.029-16.97z"
+                />
+              </svg>
+               
+              Copy to clipboard
+            </button>
+          </div>
+        </div>
+      </div>
+      <div
+        className="pf-c-code-block__content"
+      >
+        <pre
+          className="pf-c-code-block__pre"
+        >
+          <code
+            className="pf-c-code-block__code"
+          >
+            <small
+              className="textMessage"
+              data-pf-content={true}
+              id="applying-devfile-error"
+              style={
+                Object {
+                  "whiteSpace": "nowrap",
+                }
+              }
+            >
+              Failed to request factory resolver.
+            </small>
+          </code>
+        </pre>
+      </div>
+    </div>
+  </small>
+  <small
+    className=""
+    data-pf-content={true}
+  >
+    If you continue it will be ignored and a regular workspace will be created. You will have have chance to fix the Devfile from the remote IDE once the dev environment is started.
```

Comment:
```suggestion
    If you continue it will be ignored and a regular workspace will be created. You will have chance to fix the Devfile from the remote IDE once the dev environment is started.
```